packages/core/src/App/Containers/RealAccountSignup/add-or-manage-accounts.jsx

```diff
@@ -61,13 +61,12 @@ const AddOrManageAccounts = props => {
         setActiveIndex(index);
     };
 
-    const manageOrChangeAccount = (obj, setSubmitting) => {
+    const manageOrChangeAccount = currency_object => {
         setLoading(true);
-        Object.entries(obj).map(([key, value]) => {
-            if (key === 'fiat') {
+        Object.entries(currency_object).map(([key, value]) => {
```

Comment:
i am a bit skeptical about this code. apart from key 'fiat', the key here can be 'currency' as well. the refactored code was failing intermittently. just to ensure there are no logical changes, i am not changing this.

Reply:
during TS migration of its related components, we will get clarity on the possible keys and values 🧷